src/renderers/shaders/ShaderChunk/shadowmap_pars_fragment.glsl

```diff
@@ -1,12 +1,17 @@
 #ifdef USE_SHADOWMAP
 
+	uniform int isShadowCube[ MAX_SHADOWS ];
+	uniform samplerCube shadowCube[ MAX_SHADOWS ];
 	uniform sampler2D shadowMap[ MAX_SHADOWS ];
 	uniform vec2 shadowMapSize[ MAX_SHADOWS ];
 
 	uniform float shadowDarkness[ MAX_SHADOWS ];
 	uniform float shadowBias[ MAX_SHADOWS ];
 
+	uniform vec3 shadowLightPosition[ MAX_SHADOWS ];
+
 	varying vec4 vShadowCoord[ MAX_SHADOWS ];
```

Comment:
@bhouston Yes, kind-of #7037...

@mkkellogg That sounds great! I moved the light positions to view space just very recently (see #7028) and there were no references in the shadow mapping code... Is it possible to get away without altogether perhaps? Either way, no problem when reusing the lighting info...